cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -442,3 +444,60 @@ func writeInitramfsMountUnit(what, where string, utype unitType) error {
 	linkPath := filepath.Join(wantsDir, unitFileName)
 	return os.Symlink(filepath.Join("..", unitFileName), linkPath)
 }
+
+// See comment in snapd/core-initrd/latest/factory/usr/lib/systemd/system/sysroot.mount
+// before modifying the dependencies.
+const sysrootMountUnitTmpltTxt = `[Unit]
+DefaultDependencies=no
```

Comment:
I think we should add `Before=` and `Conflicts=` to `umount.target` at least. Did not we have it originally?